src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj

```diff
@@ -19,6 +19,7 @@
 	</PropertyGroup>
 	
 	<ItemGroup>
+		<!-- TODO: Add missing images? -->
```

Comment:
Which images are missing ?

Reply:
![image](https://user-images.githubusercontent.com/57812285/215865930-e55ad205-0726-4449-a102-a09a96b0f85c.png)

Reply:
Will be taking a look at this later for cleaning purposes.